views/sign-out.html

```diff
@@ -66,6 +66,7 @@
         classes: "govuk-visually-hidden"
       }) }}
 
+      {% if saveAndResume %}
       <details class="govuk-details" data-module="govuk-details" data-event-id="find-saved-information-details">
```

Comment:
Please could you indent lines 70 to 87 by two spaces, just to improve readability.